### PR TITLE
[CDAP-13157] Add a case study to the Ranger docs

### PR DESCRIPTION
**Pull request by anew**
Please click "View" on the cdap-ranger/README.rst to see the doc including images.

### CODE REVIEW

File: cdap-ranger/README.rst [hunk 1]
```diff
@@ -216,7 +215,158 @@ expect wildcard ``*`` to match 0 characters (absence of value) so a ``*`` should
...
+
+Consider a common use case for secure environments, especially data lakes:
+
+- There are different "categories" of data. For example, click events, financial data, operational metrics, etc.
+- Users typically have access to some but not all data. Therefore, data that is commonly accessed together,
+  is grouped into a category, for example, ``finance``.
+- All data in a category is owned by a headless service account, and access to the data is given through group
+  permissions or similar, more coarse, privilege roles.
+- Nobody ever logs on as the headless service account.
+- The applications and pipelines that create and process thw data are managed by Operators, that is,
```
Comment (rohitsinha54) [inline cdap-ranger/README.rst]: process the

File: cdap-ranger/README.rst [hunk 2]
```diff
@@ -216,7 +215,158 @@ expect wildcard ``*`` to match 0 characters (absence of value) so a ``*`` should
...
+
+   .. image:: _images/ranger-case-study-create-ns4.png
+
+4. Now let’s create a pipeline. Without additional policies in Ranger, this will fail:
+
+   .. image:: _images/ranger-case-study-deploy-fail1.png
+
+   We are required to give Alice ``ADMIN`` privileges on the pipeline applications she deploys.
+
+   Note that in CDAP, am application is a group of programs that logically belong together.
```
Comment (rohitsinha54) [inline cdap-ranger/README.rst]: an application

File: cdap-ranger/README.rst [hunk 3]
```diff
@@ -216,7 +215,158 @@ expect wildcard ``*`` to match 0 characters (absence of value) so a ``*`` should
...
+   A pipeline is an application with the same name as the pipeline. It contains the data pipeline
+   workflow and the MapReduce or Spark programs that execute the pipeline. For deploying the pipeline,
```
Comment (rohitsinha54) [inline cdap-ranger/README.rst]: Workflow

File: cdap-ranger/README.rst [hunk 4]
```diff
@@ -216,7 +215,158 @@ expect wildcard ``*`` to match 0 characters (absence of value) so a ``*`` should
 example to grant a user privilege on all ``programs`` the wildcard value should be as shown below.
 
 .. image:: _images/policy_management_wildcard.png
-:align: center
+  :align: center
+
+A Case Study
+============
```
Comment (rohitsinha54) [inline cdap-ranger/README.rst:221]: Maybe this ----------

Just like how it is for Conclusion section

**Comment by rohitsinha54** [comment]
Just few comments on typos. LGTM. 

Although, please note that this need changes in cdap documentation too get pulled into cdap docs.

**Review by rohitsinha54 — APPROVED** [review]
Just few comments on typos. LGTM. 

Although, please note that this need changes in cdap documentation too get pulled into cdap docs.

**Comment by anew** [comment]
I think this gets pulled in already.

**Comment by anew** [comment]
@rohitsinha54 I can't see your comments regarding typos.

**Comment by anew** [comment]
oh never mind, I had to reload.

File: cdap-ranger/README.rst (hunk 4 above)
Comment (anew) [reply cdap-ranger/README.rst:221]: That is actually a top-level heading. Whereas Conclusion is a subsection of this.

**Comment by anew** [comment]
Addressed comments and merging

**Pull request merged**